Menus/RadialMenu/RadialMenu.cs

```diff
@@ -76,6 +76,7 @@ void Start()
 			m_RadialMenuUI.actions = menuActions;
 			m_RadialMenuUI.Setup();
 			m_RadialMenuUI.visible = m_Visible;
+			m_RadialMenuUI.rayOrigin = rayOrigin;
```

Comment:
@amirebrahimi I believe it was called in RadialMenu's Start() function; after instantiating the UI.